.travis.yml

```diff
@@ -11,7 +11,7 @@ android:
       - extra-android-m2repository
 sudo: false
 before_install:
-  - nvm install 10.16.3
+  - nvm install 12.2.0
```

Comment:
Node 10 for some reason errored on `import 'whatwg-fetch'`, but Node 12 didn't. The README says the Node dependency is "latest LTS," which I guess is 14.x, but I'm hoping the latest 12.x is OK here.